### PR TITLE
Update localization.json

### PR DESCRIPTION
**Pull request by Scorprox**

### CODE REVIEW

File: EDEngineer/Resources/Data/localization.json [hunk 1]
```diff
@@ -1718,12 +1718,6 @@
       "de": null,
       "fr": null,
       "ru": null
-    },
```
Comment (msarilar) [inline EDEngineer/Resources/Data/localization.json]: good job ! just please do not remove that last block, I added it a short while ago (it was the last part of the application that it wasn't yet possible to translate)

Comment (Scorprox) [inline EDEngineer/Resources/Data/localization.json]: Hi. Sorry, I didn´t mean to do that. I dont know this github system, I am just used to Tortoise/subversion.. Do I have to fix it still?

Comment (msarilar) [inline EDEngineer/Resources/Data/localization.json]: yes you just need to add back these lines you delete (the block "Select Language")

**Comment by msarilar** [comment]
🎉 thank you !

**Pull request merged**